Xamarin.Forms.Platform.UAP/LabelRenderer.cs

```diff
@@ -35,16 +35,20 @@ public class LabelRenderer : ViewRenderer<Label, TextBlock>
 		SizeRequest _perfectSize;
 		bool _perfectSizeValid;
 
-		protected override AutomationPeer OnCreateAutomationPeer()
-		{
-			// We need an automation peer so we can interact with this in automated tests
-			if (Control == null)
-			{
-				return new FrameworkElementAutomationPeer(this);
-			}
-
-			return new FrameworkElementAutomationPeer(Control);
-		}
+		//TODO: We need to revisit this later when we complete the UI Tests for UWP.
```

Comment:
Do we have an issue in GH for this problem (so we can track it and someone will address it later)?

Reply:
https://github.com/xamarin/Xamarin.Forms/issues/3022